libcaf_io/src/ip_endpoint.cpp

```diff
@@ -209,12 +209,20 @@ std::string host(const ip_endpoint& ep) {
   switch(ep.caddress()->sa_family) {
     case AF_INET:
       inet_ntop(AF_INET,
+#ifdef CAF_WINDOWS
```

Comment:
A `const_cast` doesn't impose any runtime cost. Only Windows requires a mutable pointer, but it doesn't hurt on other OSes. So, please remove the `ifdef` trees, in both cases keeping the `const_cast` version.

Reply:
Done!